### DISCUSSION TITLE
File Size Limits

### DISCUSSION THREAD

**Discussion by adithyaamara** [python-telegram-bot/python-telegram-bot#4042] [Q&A] [answered]
What are the current file size limits that telegram can handle via this API?

One weird problem that i am facing right now is, file upload of upto 40 to 50 MB happens successfully but download for the same file fails with error "file is too big." 

Why does the download only fails if upload of same file succeeded?

PS: The largest file I managed to download is 13M, largest I managed to upload is 43 M.

**Comment by Bibo-Joshi** [comment, marked as answer]
* https://core.telegram.org/bots/api#sending-files
* https://core.telegram.org/bots/api#getfile
* https://core.telegram.org/bots/api#using-a-local-bot-api-server
* https://docs.python-telegram-bot.org/en/stable/telegram.constants.html#telegram.constants.FileSizeLimit

**Reply by adithyaamara** [reply]
@Bibo-Joshi Thank you so much. You not only answered my question but also provided me a path forward.
